<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cadastro</class>
 <widget class="QWidget" name="Cadastro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1">
        <widget class="QLineEdit" name="caixa_Categoria"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_Dep">
         <property name="text">
          <string>Seção:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="caixa_Depart"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="caixa_Produto"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="caixa_Prod">
         <property name="text">
          <string>Produto:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="caixa_Base"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="text_base">
         <property name="text">
          <string>Base:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="bt_Secao">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="text_categ">
         <property name="text">
          <string>Categoria:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="caixa_Secao"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="text_Depart">
         <property name="text">
          <string>Departamento:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="caixa_Filial"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="bt_Filial">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="bt_Categ">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="bt_Dep">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="text_Filial">
         <property name="text">
          <string>Filial:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="bt_Produto">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Cadastro Base/ Margem Cliente</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QPushButton" name="bt_Salvar">
         <property name="text">
          <string>Salvar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="bt_Limpar">
         <property name="text">
          <string>Limpar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bt_Limpar</sender>
   <signal>clicked()</signal>
   <receiver>caixa_Filial</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_Limpar</sender>
   <signal>clicked()</signal>
   <receiver>caixa_Depart</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_Limpar</sender>
   <signal>clicked()</signal>
   <receiver>caixa_Secao</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_Limpar</sender>
   <signal>clicked()</signal>
   <receiver>caixa_Categoria</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_Limpar</sender>
   <signal>clicked()</signal>
   <receiver>caixa_Produto</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_Limpar</sender>
   <signal>clicked()</signal>
   <receiver>caixa_Base</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
